From f56b668074df79748142f1d1018f0b78424c0e80 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Mon, 26 Jan 2015 22:37:59 -0500
Subject: [drm] i915: further quiet i915

Message-id: <1422311879-28096-1-git-send-email-rclark@redhat.com>
Patchwork-id: 103582
O-Subject: [RHEL7 drm] drm/i915: further quiet i915
Bugzilla: 1163074
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1163074
Upstream: n/a

The change in 702133444 muffled i915's mostly harmless chatter, but a
KERN_ERR print is still enough to (for example) interrupt the splash to
x11 handover.  This patch, in effect, changes those prints to KERN_DEBUG
(but without needing to enable drm.debug on kernel cmdline), so the
messages will still show up in dmesg if we need it to help debug other
issues.  But won't interrupt the user otherwise.

Signed-off-by: Rob Clark <rclark@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/i915/i915_drv.h b/drivers/gpu/drm/i915/i915_drv.h
index 5e3020c..a4e3208 100644
--- a/drivers/gpu/drm/i915/i915_drv.h
+++ b/drivers/gpu/drm/i915/i915_drv.h
@@ -68,7 +68,7 @@
   if (i915.verbose_state_checks)    \
    __WARN_printf(format);    \
   else        \
-   DRM_ERROR(format);    \
+   drm_ut_debug_printk(__func__, format);  \
  }        \
  unlikely(__ret_warn_on);     \
 })
@@ -79,7 +79,7 @@
   if (i915.verbose_state_checks)    \
    __WARN_printf("WARN_ON(" #condition ")\n"); \
   else        \
-   DRM_ERROR("WARN_ON(" #condition ")\n");  \
+   drm_ut_debug_printk(__func__, "WARN_ON(" #condition ")\n"); \
  }        \
  unlikely(__ret_warn_on);     \
 })
-- 
1.7.1